src/bindings/c/ov/tests/ov_c_api_test.cpp

```diff
@@ -291,6 +296,7 @@ TEST_P(ov_core, ov_compiled_model_export) {
     ov_core_free(core);
 }
 
+#ifdef BUILD_OPENCV_DEPENDENT_TESTS
```

Comment:
If we change `uchar` to `unsigned char` I think we can build this test case without opencv